src/main/java/org/alliancegenome/curation_api/jobs/executors/AgmExecutor.java

```diff
@@ -48,18 +49,19 @@ public void execLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 			agmIdsBefore.addAll(affectedGenomicModelService.getIdsByDataProvider(dataProvider.name()));
 			Log.debug("runLoad: Before: total " + agmIdsBefore.size());
 		}
-		
+
 		bulkLoadFile.setRecordCount(agms.size() + bulkLoadFile.getRecordCount());
 		bulkLoadFileDAO.merge(bulkLoadFile);
-		
+
 		BulkLoadFileHistory history = new BulkLoadFileHistory(agms.size());
 		createHistory(history, bulkLoadFile);
 		boolean success = runLoad(affectedGenomicModelService, history, dataProvider, agms, agmIdsLoaded);
-		if(success && cleanUp) runCleanup(affectedGenomicModelService, history, bulkLoadFile, agmIdsBefore, agmIdsLoaded);
+		if(success && cleanUp) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.